ux/designs/integration_details/integration_details_page.md

```diff
@@ -0,0 +1,115 @@
+# Integration Details Page
+
+The designs covers the following user interactions:
+- Integration Details Page basic layout
+  - Expandable row for integration history (versions)
+  - Kebab menu
+  - Integration visualization (normal vs. many steps/connections)
+- Creating/Replacing integration draft and publish a version from integration history
+  - Status indicators at row level
+  - Having notifications in place for publish action
+  - Modal for confirming user actions
+
+## Overview
+
+### Integration Details Page
+
+#### No Draft  
+
+![no draft section](img/integration_details_page_rework_11.16_no_draft.png)
+
+1. If there’s no draft for an integration. The draft section would not be shown on this page.
+
+2. Users would be able to “Create Draft” from the current version.
+
+#### Basic layout
+
+![basic layout](img/integration_details_page_rework_11.16_genericlayout.png)
+
+1. Moved status label to the top section right next to Integration Name. Using “check mark” icon to indicate this integration is active.
+
+2. Moving Draft out of the integration history. Users would be able to “Publish” and “Edit” the draft.
+
+3. Added row indicator (green bar) and checkmark icon to indicate active and running version. For active version, users would only be able to perform “Replace Draft” with the active version.
+
```

Comment:
For symmetrical reasons (e.g. to have a kebab menu on all rows), what's about to move the "Stop Integration" and the "Replace Draft" integration into the kebab menu for an active integration ? (So 'Stop integration' is the opposite of 'Publish' in the other integrations ?

Reply:
OK, @dongniwang and I agree; "Replace Draft" and "Stop Integration" will be kebab items for the current version.